crates/mdman/tests/compare/expected/options.md

```diff
@@ -37,7 +37,7 @@ A description of the command.
 <dd class="option-desc">This has multiple flags.</dd>
 
 
-<dt class="option-term" id="option-options-named-arg..."><a class="option-anchor" href="#option-options-named-arg..."></a><em>named-arg...</em></dt>
+<dt class="option-term" id="option-options-named-arg…"><a class="option-anchor" href="#option-options-named-arg…"></a><em>named-arg…</em></dt>
```

Comment:
> * I don't think fancy ellipses in HTML attributes in Markdown is correct:
> 
>   ```
>   <dt class="option-term" id="option-options-named-arg…"><a class="option-anchor" href="#option-options-named-arg…"></a><em>named-arg…</em></dt>
>   ```


Since Cargo itself doesn't use this kind of flag, I feel it's fine to have this, though it becomes more subtle to debug and also hard to predict the actual link from raw content 🥲.